third_party/contour/gateway-internal.yaml

```diff
@@ -15,35 +15,14 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  labels:
-    control-plane: contour-operator
   name: contour-internal
 ---
-apiVersion: operator.projectcontour.io/v1alpha1
-kind: Contour
-metadata:
-  name: contour-internal
-  namespace: contour-internal
-spec:
-  gatewayControllerName: projectcontour.io/contour-internal/contour
-  namespace:
-    name: contour-internal
-  ingressClassName: contour-internal
-  networkPublishing:
-    envoy:
-      containerPorts:
-      - name: http
-        portNumber: 8081
```

Comment:
@dprotaso @nak3 how was this used? Seems like the gateway below just used port 80?

Not sure if we can define this using the gateway-provisioner. If this is needed, we probably need to use helm to have more configuration flags.

Reply:
@KauzClay

Reply:
IIRC, the `8081` port is used for prober. The current implementation lists endpoint IP which is `8081`, and probe against it.
If it works with private (cluster local) service works without `8081`, I think we can drop it, though.